src/Mono.Android/Android.Runtime/JNINativeWrapper.cs

```diff
@@ -20,6 +20,12 @@ static void get_runtime_types ()
 				"Mono_UnhandledException", BindingFlags.NonPublic | BindingFlags.Static);
 			if (mono_unhandled_exception_method == null)
 				AndroidEnvironment.FailFast ("Cannot find System.Diagnostics.Debugger.Mono_UnhandledException");
+#endif
+#if NETCOREAPP
+			mono_unhandled_exception_method = typeof (Android.Runtime.JNIEnv).GetMethod (
```

Comment:
Instead of duplicating this logic, we should instead make `JNIEnv.mono_unhandled_exception` *`internal`*, so that we can directly access it here:

```csharp
mono_unhandled_exception_method = JNIEnv.mono_unhandled_exception.Method;
```

Reply:
Fixed :)